### ISSUE TITLE
clang-format crashes with "QualifierAlignment: Left" on "const volatile" method declarations

### ISSUE THREAD

**Issue by AMoe-LACOS** [llvm/llvm-project#56696]
Minimal example source file to reproduce the issue

```cpp
class Queue
{
    bool empty() volatile const
    {
        return true;
    }
};
```

[crash_example.zip](https://github.com/llvm/llvm-project/files/9178323/crash_example.zip)

[configuration.zip](https://github.com/llvm/llvm-project/files/9178334/configuration.zip)

clang-format version 14.0.6 (Windows 64bit)

Stacktrace:
```console
0x00007FF6F84B7D4E (0x0000000000000000 0x0000000000000000 0x0000002467589E90 0x0000002467589EB0)
0x00007FF6F84B86CE (0x0000022140790000 0x00000221407EF398 0x0000000000000000 0x0000000300000000)
0x00007FF6F84C2519 (0x0000000000000000 0x00007FFC299BDF4C 0x000000246758AD60 0x000000246758B0F0)
0x00007FF6F84B8E52 (0x0000000000000000 0x0000000000000001 0x00007FF600000000 0x0000022140981140)
0x00007FF6F84B608B (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000)
0x00007FF6F84C2519 (0x0000F825A5F345FF 0x0000000000000051 0x0000000000000000 0x000000246758CC30)
0x00007FF6F84B29CF (0x0000000000000D93 0x000000000000021B 0x00000221407B20A0 0x0000000000000099)
0x00007FF6F849DD6E (0x00000221407A39C0 0x00007FFC299B47B1 0x0000000000000000 0x0000022140790000)
0x00007FF6F849EC69 (0x000002214079F62C 0x0072006700000000 0x000002214079F600 0x00000000002C002A)
0x00007FF6F842545C (0x0000000000000001 0x0000022140790000 0x0000000000000101 0x00007FFC299BB44D)
0x00007FF6F84236F6 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000)
0x00007FF6F85A43D8 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000)
0x00007FFC28CE7034 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000), BaseThreadInitThunk() + 0x14 bytes(s)
0x00007FFC299E2651 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000), RtlUserThreadStart() + 0x21 bytes(s)
```

**Comment by mydeveloperday** [comment]
possible fix https://reviews.llvm.org/D131360

**Comment by owenca** [comment]
It seems that this has already been fixed by 492cb7b.
